server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java

```diff
@@ -117,7 +117,7 @@
         false,
         true,
         RestStatus.TOO_MANY_REQUESTS,
-        EnumSet.of(ClusterBlockLevel.METADATA_WRITE, ClusterBlockLevel.WRITE)
```

Comment:
I think the [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/7.16/index-modules-blocks.html#index-block-settings) hints that metadata writes are also blocked, can we make that clearer too?

Reply:
Yep, fixed in e8bfd362a14af050a1fb95b7867bfe2b47d9f2c9.